html/semantics/scripting-1/the-script-element/README.md

```diff
@@ -0,0 +1,16 @@
+# Script tests
+
+## Import attributes & JSON/CSS modules
+
+The import attributes proposal changed the keyword from `assert` to `with`, after that it was already been implemented and shipped in some browsers. Thus, there are some implementations that only support the `assert` syntax and others that only support the `with` syntax.
+
+For this reason, the import attributes, JSON modules and CSS modules are duplicated to use both keywords:
+| `with` keyword        | `assert` keyword           |
+|:----------------------|:---------------------------|
+| `./import-attributes` | `./import-assertions`      |
+| `./json-module`       | `./json-module-assertions` |
+| `./css-module`        | `./css-module-assertions`  |
+
+All changes in one folder should be reflected in the corresponding folder, because the two syntaxes have the same semantics.
+
+Once there are no more implementations that support `assert` without supporting `with`, the `assert` folders can be deleted. This is expected to happen no later than the proposal reaching Stage 4 in the TC39 process.
```

Comment:
This is not the correct criteria. The correct criteria is when we've determined that `assert` is not required for web compatibility, and then proceed to remove it from the spec.

In other words, if it's in the spec, it needs to be tested; if it's not in the spec, it doesn't.

Reply:
Even if it's just a syntactic alias for with? I assumed it was similar to how, for example, wpt doesn't need to test what `document["getElementById"]` and `document?.getElementById` do if it's already testing `document.getElementById`.

Reply:
We need test coverage for both paths (like we do for multiple types of JS property access). Whether we need a full duplicate test suite is debatable, but it shouldn't be possible to get 100% without implementing `with`, while `with` is in the spec.

Reply:
No, but it should be possible to get 100% without implementing assert even if assert is in the spec, because it's marked as "optional, do not implement this unless you really need to" in the spec.

Anyway, this?
> The web compatibility of removing the `assert` keyword is being investigated. If it will be deemed feasible, it will be removed from the proposal and the `assert` tests can be deleted.

Reply:
Sorry, I misspoke and meant to write `assert`. I'm not sure I agree with that working mode, but regardless, your new wording SGTM.

Reply:
Readme updated!